website/content/specification/mapping.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Binding to XML, JSON, and YAML"
+linkTitle: Format Bindings
+description: "Discussion of how Metaschema primitives map to XML, JSON, and YAML primitives"
+weight: 50
+aliases:
+- /specification/mapping/
+---
+
+# Binding to XML, JSON, and YAML
+
+**Note: This section of the specification is still a work in progress.**
+
+XML, JSON, and YAML each use specialized terminology and format primitives. As a notation for an object-based data, YAML is fairly similar to JSON, while XML is quite different to the other two. While all data format describe tree structures (directed graphs), each format (with its implicit data model) has its peculiar design, which requires specification in detail.
```

Comment:
Minor, non-blocking: I would say particular, not peculiar. The latter sounds like we want to and can make a value judgement (that's my personal take), feel free to ignore.

```suggestion
XML, JSON, and YAML each use specialized terminology and format primitives. As a notation for an object-based data, YAML is fairly similar to JSON, while XML is quite different to the other two. While all data format describe tree structures (directed graphs), each format (with its implicit data model) has its particular design, which requires specification in detail.
```